### PR TITLE
Update best practices on supplying images

### PR DESCRIPTION
**Pull request by lsauser**
testing just one update, since it's my first! changed "apps" --> "extensions" in one instance.

### CODE REVIEW

**Comment by lsauser** [comment]
@googlebot I signed it!

File: site/en/docs/webstore/images/index.md [hunk 1]
```diff
@@ -5,28 +5,33 @@ date: 2018-06-11
...
 - [Screenshots][5]
 
-Only the app icon, a small promotional image, and a screenshot are mandatory. However, providing
-attractive versions of both required and optional images increases your app's chances of getting
-noticed. For example, your app can't be featured unless you provide a large or marquee promotional
-image.
+Only the extension icon, a small promotional image, and a screenshot are mandatory. However, providing
+attractive versions of both required and optional images increases your extension's chances of getting
+noticed. For example, your extension can't be featured in marquee unless you provide a marquee promotional
+image. Follow the below image best practices to increase your chance of being featured.
```
Comment (awfuchs) [inline site/en/docs/webstore/images/index.md:18]: Not a fan of "the below &lt;noun&gt;". Consider something like...
"Follow the below image best practices" --> "Observe the following image best practices"

Or perhaps:
Follow our [image best practices] to increase...
.
.
.
[image best practices]: #image-best-practices

Comment (lsauser) [reply site/en/docs/webstore/images/index.md]: done!

**Comment by lsauser** [comment]
@googlebot I signed it!

File: site/en/docs/webstore/images/index.md (hunk 1 above)
Comment (awfuchs) [reply site/en/docs/webstore/images/index.md]: Line 318 isn't right. The generated anchor for that heading is `image-best-practices`, not `imagebestpractices`.

**Review by awfuchs — APPROVED** [review]

**Pull request merged**